scripts/ErrorReporter/errorreport.py

```diff
@@ -108,8 +108,8 @@ def get_plain_text_edit_field(self, text_edit, expected_type):
 
     def check_placeholder_text(self):
         if not self.free_text_edited:
-            self.input_free_text.setPlainText("")
             self.free_text_edited = True
+            self.input_free_text.setPlainText("")
```

Comment:
This looks like it is because Qt4 doesn't have the `setPlaceHolderText` option so mimics it with this function and calling set again calls this unless `free_text_edited` is True.